app/views/search/index.html.erb

```diff
@@ -100,10 +104,16 @@
                              url_for(@search_request.to_params(order_target: "score"))) %>
         </li>
         <li>
-          <%= link_to_unless(@search_request.order_target == "date",
-                             tag.i(class: "fas fa-clock") + " " +
-                               l(:label_full_text_search_order_target_date),
-                             url_for(@search_request.to_params(order_target: "date"))) %>
+          <%= link_to_unless(@search_request.order_target == "last_modified_time",
```

Comment:
```suggestion
          <%# TODO: Add a comment why "date" is here %>
          <%= link_to_unless(@search_request.order_target.include?(["date", "last_modified_time"],
```

Reply:
fix: [61f2096](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/61f20967cf7d53513c6f1314c37eba69cdf6135d)
Thanks, I missed it.